packages/components/src/components/notification/notification.tsx

```diff
@@ -164,24 +170,22 @@ export class Notification {
             `variant-${this.variant}`,
             this.isOpen && 'open'
           )}
-          role={this.role}
+          role={this.innerRole}
         >
           <div part="icon" aria-hidden="true">
             <slot name="icon">
               <IconTag size={ICON_SIZE} selected={this.type === 'toast'} />
             </slot>
           </div>
-          <div
-            part="body"
-            aria-live={this.role === undefined ? undefined : this.innerAriaLive}
-          >
+          <div part="body" role={this.innerRole} aria-live="polite">
```

Comment:
Should this `div` really have the same role as its parent element? And setting `aria-live="polite"` may not always be correct (if `this.innerRole === 'alert'`). I'd suggest that:

- Only the parent element has the proper ARIA role, i.e. `role={this.innerRole}` (already implemented)
- Only the parent element has the attribute `aria-live` depending on `this.innerRole`. If role is `'alert'` the attribute value is `assertive`. Otherwise, the attribute value is `polite`.

Reply:
i removed this